.gitmodules

```diff
@@ -124,6 +124,9 @@
 [submodule "projects/multicenter-phantom"]
 	path = projects/multicenter-phantom
 	url = https://github.com/conpdatasets/multicenter-phantom.git
+[submodule "projects/Bigbrain_3DROIS"]
```

Comment:
In the git submodule, the s after ROI is upper case but in the actual directory it is lower case.